src/Reflection/ReflectionProperty.php

```diff
@@ -118,6 +119,33 @@ private function getVisibilityAsString()
     }
 
     /**
+     * Set the default visibility of this propery. Use the core \ReflectionProperty::IS_* values as parameters, e.g.:
+     *
+     * @param int $newVisibility
+     * @throws \InvalidArgumentException
+     */
+    public function setVisibility($newVisibility)
+    {
+        $this->node->type = ~Class_::MODIFIER_PRIVATE;
```

Comment:
Hmm, no; tests failing using `$this->node->type &= ~Class_::MODIFIER_PRIVATE;` etc.